src/components/menu/menu.stories.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+
+  import Menu from './menu.svelte'
+  import Icon from '../icon/icon.svelte'
+
+  let button
+</script>
+
+<Meta title="Menu" component={Menu} />
+
+<Template let:args>
+  <div class="container">
+    <button bind:this={button}> View page source </button>
+    <Menu isOpen={true} target={button}>
+      <leo-menu-item>
+        <div>Regenerate</div>
+      </leo-menu-item>
+      <leo-menu-item> Copy </leo-menu-item>
+      <leo-menu-item> Share </leo-menu-item>
+      <leo-menu-item>
+        <div class="item">
+          <div>New Chat</div>
+          <Icon name="plus-add" />
+        </div>
+      </leo-menu-item>
+    </Menu>
+  </div>
+</Template>
+
+<Story name="Default" />
```

Comment:
Probably worth adding a `slots` story, for completeness

Reply:
added